pyproject.toml

```diff
@@ -80,8 +80,8 @@ dependencies = [
   "semver~=3.0.1",
   "send2trash",
   "test-tube~=0.7.5",
-  "torch~=2.0.1",
-  "torchvision~=0.15.2",
+  "torch~=2.1.0",
+  "torchvision~=0.16",
```

Comment:
@lstein are we okay with these changes - are you aware of any compatibility issues? this "works for me"™ on Linux both under Docker and bare-metal, but I haven't tested any other configurations.

Reply:
@lstein - testing, this seems tow ork for me as well. going to approve